tests/deploy.py

```diff
@@ -68,22 +68,22 @@ def juju_deploy(
     path = tempfile.mkdtemp()
     rsync(charm_source, path)
     args = ['deploy', '--series', series]
-    if service_name is None:
-        service_name = charm_name
+    if app_name is None:
+        app_name = charm_name
     if options is not None:
-        config_file = make_charm_config_file({service_name: options})
+        config_file = make_charm_config_file({app_name: options})
```

Comment:
Has this change been performed in this function? I don't see the change elsewhere.